packages/firestore/src/core/view.ts

```diff
@@ -275,13 +275,16 @@ export class View {
    *        this change.
    * @param targetChange - A target change to apply for computing limbo docs and
    *        sync state.
+   * @param waitForRequeryResult - Whether the target is pending to run a full
```

Comment:
nit: Update the documentation for the `waitForRequeryResult` parameter to indicate what it means if the parameter is omitted (i.e. it is treated the same as if `false` were specified).